packages/react-scripts/scripts/start.js

```diff
@@ -60,6 +60,11 @@ if (isSmokeTest) {
   };
 }
 
+// Used to detect flow errors we want to swallow
+function isFlowIntegrationErrorMessage(message) {
```

Comment:
yep, that's the kind of feedback I was expecting... I'll see how I can hide that... Maybe in `formatWebpackMessages`?